pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -1630,7 +1630,7 @@ public void setSchemaCompatibilityStrategy(
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
             @ApiResponse(code = 404, message = "Namespace doesn't exist"),
             @ApiResponse(code = 409, message = "Concurrent modification") })
-    public boolean getIsAllowAutoUpdateSchema(
+    public Boolean getIsAllowAutoUpdateSchema(
```

Comment:
Maybe we need to think about why the ``internalGetIsAllowAutoUpdateSchema`` method returns null , because null is an incorrect result.

Reply:
default value in policies is null. beacuse

Reply:
@gaozhangmin

same to above, why return null if an api needs to return a boolean.

Reply:
default value in policies is null, if you don't set this value.

Reply:
In my opinion, we need to return a boolean when we define an api to return a boolean. If not, it is an exception state.

What do you think about this?

Reply:
No, `bin/pulsar-admin namespaces get-deduplication public/default` 
this command would print value null if default value is null. I referenced this. I think it's Reasonable.

Reply:
@gaozhangmin  Ok, i see.